conns.sh

```
@@ -0,0 +1,182 @@
+./c1/1.lox -> ./if/if.lox
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)